functions/responsestreaming/StreamBigQuery.Tests/StreamBigQueryTest.cs

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+using Google.Cloud.Functions.Testing;
+using System.Net;
+using System.Net.Http;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace StreamBigQuery.Tests;
+
+public class StreamBigQueryTest : FunctionTestBase<StreamBigQuery.Function>
+{
+    [Fact]
+    public void TestRunStreamBigQuery()
+    {
+        var request = new HttpRequestMessage
+        {
+            Method = HttpMethod.Post,
+            Content = new StringContent("")
+        };
+
+        ExecuteHttpRequestAsync(request, async response =>
+        {
+            Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+            var actualContent = await response.Content.ReadAsStringAsync();
+            var outputLines = actualContent.Split(new[] { '\n' });
+            int rowCount = outputLines.Count();
+            Assert.Equal(1000, rowCount);
+        });
+}
```

Comment:
It looks like this code wouldn't compile at the moment - isn't it missing a closing brace?